lib/load.py

```diff
@@ -31,10 +25,10 @@ def load_csv(data_resource, config={}, connection=None):
 
 
 def delete_datastore_table(data_resource, config={}):
-    header = {'Authorization': Variable.get('CKAN_SYSADMIN_API_KEY')}
+    header = {'Authorization': config['CKAN_SYSADMIN_API_KEY']}
```

Comment:
Why not pass these arguments explicitly to the function rather than pull from config

Reply:
You mean passing several args like `load.create_datastore_table(data_resource, Variable.get('CKAN_SYSADMIN_API_KEY'), Variable.get('CKAN_SITE'))` ?